README.md

```diff
@@ -18,9 +18,14 @@ A source connector pulls data from an external resource and pushes it to downstr
 
 ### Configuration
 
-| name                  | description                           | required | default value |
-|-----------------------|---------------------------------------|----------|---------------|
-| `source_config_param` | Description of `source_config_param`. | true     | 1000          |
+| name       | description                                            | required | default value |
+| ---------- | ------------------------------------------------------ | -------- | ------------- |
+| `host`     | The hostname or IP address of the MySQL server         | true     |               |
+| `port`     | The port number on which the MySQL server is listening | false    | 3306          |
+| `user`     | The MySQL user account name for authentication         | true     |               |
+| `password` | The password for the MySQL user account                | true     |               |
+| `database` | The name of the specific MySQL database to connect to  | true     |               |
+| `tables`   | The list of tables to pull data from                   | true     |               |
 
 ## Destination
```

Comment:
If destination is still not implemented, I would remove the destination section altogether and add a mention to the `Known Issues & Limitations` section. Same with the Planned work.